pkg/kv/kvserver/replica_rangefeed.go

```diff
@@ -95,10 +96,10 @@ var ErrBufferedSenderNotSupported = unimplemented.NewWithIssue(
 // unimplemented and disabled everywhere (#126560).
```

Comment:
Let's update the comment while we are here. This is no longer unimplemented :D.

Reply:
🥳 Good catch, done.